### PR TITLE
78 design all pages based on figma

### LINKED ISSUE #59: Create new Flutter widget for Flutter Object
**Short description:**
Create all widget for display all Flutter Object #34

**Describe the solution you'd like**
Create all widget : 

- [x] User
1. Possibility to display just all data alone or all information.

- [x] Service
1. Possibility to display just name or all information if it's available.

- [x] Action
1. Possibility to display just name or all information if it's available.

- [x] Reaction
1. Possibility to display just name or all information if it's available.

- [ ] Maybe Error

- [ ] Area (due to date after because I don't know what HOG want about that)

### LINKED ISSUE #78: Design all pages based on Figma
Design pages : 

- [ ] Home Page
- [ ] Create Area
- [ ] Settings Area
- [ ] Settings Pages

### LINKED ISSUE #92: Add Area Icons

### PR DESCRIPTION
**Pull request by TomDUVAL-MAHE**
Good evening, evening friends !

If you want to read it good luck otherwise you can just test it on branch #78  ;D

This Pull Request integrates the basic front of the application, it brings with it the redesign of all the pages of the application as well as some of its beautifulage elements!

This will actually have to be improved later because some things have been corrected or changed. 

Thank you for your time

Photos don't be in order 

image

image

image

image

image

image

image

image

image

image

image

image

image

Close #59
Close #78
Close #92

### CODE REVIEW

File: application/app_code/lib/flutter_objects/area_data.dart [hunk 1]
```diff
@@ -46,10 +49,116 @@ class AreaData {
...
+            Text(
+              'Description : \n\n $description',
+              style: const TextStyle(color: Colors.black),
+            )
+
+            /// Put Description when it's in DB
+          ],
+        ));
+
+        ///        listDisplay.add(Text(action.name));
```
Comment (tbellicha) [inline application/app_code/lib/flutter_objects/area_data.dart]: Why don't you remove these comments ?

File: application/app_code/lib/flutter_objects/area_data.dart [hunk 2]
```diff
@@ -74,19 +183,79 @@ class AreaData {
...
```
Comment (tbellicha) [inline application/app_code/lib/flutter_objects/area_data.dart:254]: same

File: application/app_code/lib/flutter_objects/area_data.dart (hunk 1 above)
Comment (TomDUVAL-MAHE) [reply application/app_code/lib/flutter_objects/area_data.dart]: :+1:

File: application/app_code/lib/flutter_objects/area_data.dart (hunk 2 above)
Comment (TomDUVAL-MAHE) [reply application/app_code/lib/flutter_objects/area_data.dart:254]: :+1:

**Review by tbellicha — APPROVED** [review]

**Review by LucasTesnier — APPROVED** [review]
AmAzing

**Pull request merged**